WPMedia/arc-themes-components
# updated headphones icon to use themes v1 path

## Ticket

- [THEMES-1838](https://arcpublishing.atlassian.net/browse/THEMES-1838)

## Description

_Information about what you changed for this PR_

## Acceptance Criteria

_copy from ticket_

## Test Steps

1. Checkout branch - `git checkout THEMES-1838`
2. Run fusion and verify that updated icon looks okay

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1838]: https://arcpublishing.atlassian.net/browse/THEMES-1838?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
